libs/island-ui/theme/src/lib/colors/colors.ts

```diff
@@ -0,0 +1,202 @@
+import { color } from './primitivesMapper'
+
+const {
+  primary: { blue, dark, red, white: white400 },
+  secondary: { blueberry, purple, rose },
+  accent: { mint, yellow },
+} = color
+
+// Primary colors
+export const blue600 = blue[600] ?? '#0044b3'
+export const blue400 = blue[400] ?? '#0061ff'
+export const blue300 = blue[300] ?? '#99c0ff'
+export const blue200 = blue[200] ?? '#ccdfff'
+export const blue100 = blue[100] ?? '#ccdfff'
+
+export const dark400 = dark[400] ?? '#00003c'
+export const dark350 = dark[350] ?? '#33335a'
+export const dark300 = dark[300] ?? '#9999b1'
+export const dark200 = dark[200] ?? '#ccccd8'
+export const dark100 = dark[100] ?? '#f2f2f5'
+
+export const red600 = red[600] ?? '#b30038'
+export const red400 = red[400] ?? '#ff0050'
+export const red300 = red[300] ?? '#ff99b9'
+export const red200 = red[200] ?? '#ffccdc'
+export const red100 = red[100] ?? '#fff2f6'
+
+export const white = white400[400] ?? '#ffffff'
```

Comment:
There is only one white, can this be this instead, like `black`?

```suggestion
export const white = '#ffffff'
```

Reply:
yea, totally 💯